core/document/src/color.rs

```diff
@@ -1,5 +1,10 @@
 use serde::{Deserialize, Serialize};
 
+/// Structure that represent a color\
+/// internally alpha is stored as f32 that go from 0.0 (transparent) to 1.0
+/// the other components (rgb) are stored as f32 that go from 0.0 up to f32::MAX,
+/// the value are proportinal to the number of photons hitting the camera sensor
```

Comment:
For better clarity about the concept, would you mind switching this sentence to the following?:

> the values encode the brightness of each channel proportional to the light intensity in cd/m² (nits) in HDR, and `0.0` (black) to `1.0` (white) in SDR color.